Cargo.lock

```diff
@@ -2609,7 +2622,7 @@ dependencies = [
  "cargo-emit 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "crc 1.8.1 (registry+https://github.com/rust-lang/crates.io-index)",
  "curve25519-dalek 3.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "displaydoc 0.2.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "displaydoc 0.1.7 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
This seems to be happening with all the crates in the mobilecoin submodule, even though this causes builds to (properly) fail when a displaydoc error message would be truncated...

Reply:
thank you, i will investigate

Reply:
I think i messed up the submodule when i rebased